utilities/geocoder.py

```diff
@@ -1,46 +1,85 @@
 """
 Geocoding Utility:
-
 We'll be using this: https://github.com/slimkrazy/python-google-places#python-google-places
 to geocode addresses.
 """
 import os
+import sys
+sys.path.append('../')
 
 import pandas as pd
+import sqlalchemy as sqla
+from sqlalchemy import create_engine
 
-from googleplaces import GooglePlaces, types, lang
+SBA_DWH = os.getenv('SBA_DWH')
+engine = create_engine(SBA_DWH)
 
-YOUR_API_KEY = os.getenv('GOOGLE_PLACES_API')
+from geopy import GoogleV3
+from utilities.db_manager import DBManager
 
+YOUR_API_KEY = os.getenv('GOOGLE_MAPS_API')
 
-def geocode(df, api_key=YOUR_API_KEY):
-    """Add Geocoded columns to df
+def create_address_column(df):
+    """creates a new column on the dataframe for the geocoding address  
+    Keyword Args: 
+    address: address
+    """
+    df['address'] = df.borr_street + ', ' + df.borr_city + ', ' + df.borr_state + ', ' + df.borr_zip.map(str)
+    return df
+
+def geocode_address(address, api_key=YOUR_API_KEY):
+    """Geocodes a single address 
+    Keyword Args: 
+    address: string with the address"""
 
+    geolocator = GoogleV3(api_key = api_key)
+    query_result = geolocator.geocode(address)
+    if query_result != None:
+        latitude = query_result.latitude
+        longitude = query_result.longitude
+    else:
+        latitude = "Not found"
+        longitude = "Not found"
+    return latitude, longitude
+
+def create_geocode_table(df, dbm):
+    """Add Geocoded columns to df
     Keyword Args:
     df: Dataframe which must have an "address" column with a clean address
     api_key: Google Places API Key
```

Comment:
This is no longer a keyword rag. Can you change this to `dbm` and a quick description of dbm?